src/toil_scripts/adam_pipeline/adam_preprocessing.py

```diff
@@ -37,8 +37,8 @@
 from toil.job import Job
 
 from toil_scripts.adam_uberscript.automated_scaling import SparkMasterAddress
-from toil_scripts.batch_alignment.bwa_alignment import docker_call
-from toil_scripts.spark_utils.spawn_cluster import start_spark_hdfs_cluster
+from toil_scripts.batch_alignment.bwa_alignment import docker_call, mock_mode
+from toil_scripts.spark_utils.spawn_cluster import *
```

Comment:
I doubt it.

```
hannes-mbpr:x hannes $ cat foo.py
from threading import Event
hannes-mbpr:x hannes $ cat bar.py
from multiprocessing import Event
from foo import *
assert Event.__module__ == 'multiprocessing', Event.__module__
hannes-mbpr:x hannes $ python bar.py 
Traceback (most recent call last):
  File "bar.py", line 3, in <module>
    assert Event.__module__ == 'multiprocessing', Event.__module__
AssertionError: threading
(bd2k:/__me__/)hannes-mbpr:x hannes $ 
```

This has bitten us with those dupes of the download_url function.